src/lib/containers/CardContainer.tsx

```diff
@@ -189,7 +197,8 @@ export class CardContainer extends React.Component<CardContainerProps, CardConta
 
     return (
       <div>
-        {(unitDescription && showBarChart) &&
+        {title &&  <h2 className="SRC-title">{title}</h2>}
+        {(!title && unitDescription && showBarChart) &&
           <TotalQueryResults
```

Comment:
TotalQueryResults visibility depends on title (not being set)?

Reply:
Yes, at least in the current set of mocks there's no view that has both a title and a card count